app/controllers/users_controller.rb

```diff
@@ -1,5 +1,10 @@
 class UsersController < ApplicationController
-  def index; end
+  def index
+    @recent_posts = User.all.map(&:recent_posts).flatten
```

Comment:
- [ ] We are supposed to return all users in this action
```suggestion
    @users = User.all
```